src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_28 calendar--start-day_fri">
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/224533537-12ec29bf-2f55-4600-97a2-1f056b6fdccf.png)

Reply:
it is difficult to guess what was meant here.
this code currently looks like
 `  <body>
    <div class="calendar calendar--month-length_30 calendar--start-day_thu">
      <div class="calendar__day"></div>`